esp/applications/eclwatch/application.yaml

```diff
@@ -16,4 +16,5 @@ application:
    - ws_esdlconfig
    - ws_store
    - ws_codesign
+   - ws_container
```

Comment:
It's interesting.  We use the term containerized to mean we were compiled as a docker container, but I'm not sure "container" is the right term to use in regards to getting information about our cloud environment. Not sure what the best name is though.

Reply:
Could be something similar to ws_resources.

Reply:
@jakesmith @richardkchapman @ghalliday any thoughts on the name?

Reply:
Agree, we're being a bit fast and loose with the term containerized vs cloud in places I think, but you're right it's not the right term, and particularly not right where it has external exposure.

As you suggest, it's not really pertinent that it's 'cloud' or 'containerized' from the service users point of view.
Something generic for resource discovery, like ws_resources does make sense.
I can't think of a better name. It will a repository for calls to get services, jobqueues etc.